containers/release-candidate/Dockerfile

```diff
@@ -15,20 +15,61 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.fedoraproject.org/fedora-minimal:36
+FROM registry.fedoraproject.org/fedora:36
 
 WORKDIR /root
 
-COPY setup-container.sh /usr/bin/
-RUN /usr/bin/setup-container.sh
+# Delete cruft fedora leaves in the homedir
+RUN rm /root/*
 
-# Needed to get WiX to run in wine on Linux. See wix_wine.sh for more details
-# on why we need to do this and how it works
+# Install dependencies, explicitly not enabling the updates repo so we are
+# pinned at a particular fedora release.
+RUN \
+  dnf -y --quiet --repo=fedora install \
+    clang \
+    git \
+    java-1.8.0-devel \
```

Comment:
Java 8 is pretty close to end of life.  Would Java 11 work here?

Reply:
I know of systems that unfortunately still use Java 8. I prefer we continue with this just to ensure no compatibility issues, unless there's a good reason..